### PR TITLE
[TASK] Remove no longer needed path to binaries

### PR DESCRIPTION
**Pull request by Starkmann**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -159,9 +159,9 @@
 		"ci:tests:create-directories": "mkdir -p .Build/public/typo3temp/var/tests",
 		"ci:tests:functional": [
 			"@ci:tests:create-directories",
-			"find 'Tests/Functional' -wholename '*Test.php' | parallel --gnu 'echo; echo \\\"Running functional test suite {}\\\"; .Build/bin/phpunit -c Build/phpunit/FunctionalTests.xml {}';"
+			"find 'Tests/Functional' -wholename '*Test.php' | parallel --gnu 'echo; echo \\\"Running functional test suite {}\\\"; phpunit -c Build/phpunit/FunctionalTests.xml {}';"
```
Comment (oliverklee) [inline composer.json:162]: For the cases where we are piping something to a Composer script (in this case: functional tests), we'll need to keep using the full path AFAIK.

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Review by oliverklee — CHANGES_REQUESTED** [review]
Also, in the commit message and PR title, this one here should be a `[TASK]`, not a `[FEATURE]`.

File: composer.json (hunk 1 above)
Comment (Starkmann) [reply composer.json:162]: Might be can not really test this because of parallel error

**Review by oliverklee — COMMENTED** [review]
Looks good, but needs a rebase to resolve the merge conflicts.

**Pull request merged**